src/pages/PreviousRuns.test.js

```diff
@@ -170,15 +177,17 @@ describe('Previous Runs page', () => {
 
     // Assert that sort by Status worked
     const updatedDataRow1Cells = within(rows[1]).queryAllByRole('cell')
-    expect(updatedDataRow1Cells.length).toBe(4)
+    expect(updatedDataRow1Cells.length).toBe(5)
     within(updatedDataRow1Cells[0]).getByText('b7234aae-6f43-405e-bb3a-71f924e09825')
     within(updatedDataRow1Cells[1]).getByText('Failed')
-    within(updatedDataRow1Cells[3]).getByText(/Jul 14, 2022/)
+    within(updatedDataRow1Cells[2]).getByText(/Jul 14, 2022/)
+    within(updatedDataRow1Cells[4]).getByText(/Jul 14, 2022/)
```

Comment:
Do we want to assert that the time is present and correct as well as the date?

Reply:
Asserting the timestamps is a little complicated. Since the timestamps in the data is being converted to Local timezone, it returns different time when the tests are run locally vs in GitHub action. As a result everywhere we only compare dates.